recipes/debug_assert/all/test_package/CMakeLists.txt

```diff
@@ -1,9 +1,8 @@
-cmake_minimum_required(VERSION 3.1)
-project(test_package)
+cmake_minimum_required(VERSION 3.15)
+project(test_package LANGUAGES CXX)
 
-include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
-conan_basic_setup()
+find_package(debug_assert REQUIRED CONFIG)
 
 add_executable(${PROJECT_NAME} test_package.cpp)
-target_link_libraries(${PROJECT_NAME} ${CONAN_LIBS})
+target_link_libraries(${PROJECT_NAME} PRIVATE debug_assert::debug_assert)
 set_property(TARGET ${PROJECT_NAME} PROPERTY CXX_STANDARD 11)
```

Comment:
Should I keep the `CXX_STANDARD` setting in test_package if `check_min_cppstd()` is used in the conanfile.py?
I suppose it should only be necessary if the standard needs to be lower than the compiler's default or the one set by `compiler.cppstd`?

Reply:
Yes, I think it's better keep it because we don't pass the cppstd to v1 builds, so it would fail